Sources/OrderedCollectionsDifferentiable/OrderedCollections.swift

```diff
@@ -0,0 +1,2 @@
+// Export the OrderedCollections API from Swift Collections which this module extends with `Differentiable` support
+@_exported import OrderedCollections
```

Comment:
Same comment, shouldn't this be exporting `OrderedCollectionsDifferentiable`?

Reply:
This exports the OrderedCollections module when a user imports OrderedCollectionDifferentiable. 
That way you don't have to do both everywhere you want to use them.